hack/fedora33dev/docker-dev-push.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+IMG=quay.io/dgoodwin/hive:latest
```

Comment:
Works for me, just need to drop this `IMG` override

Reply:
Updated, propose we merge and iterate if others start using.